src/EventListener/ProductTranslationUpdateListener.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Setono\SyliusRedirectPlugin\EventListener;
+
+use Doctrine\ORM\Event\LifecycleEventArgs;
+use Setono\SyliusRedirectPlugin\Model\RedirectInterface;
+use Setono\SyliusRedirectPlugin\Resolver\InfiniteLoopResolverInterface;
+use Sylius\Component\Core\Model\ProductTranslationInterface;
+use Sylius\Component\Resource\Exception\UpdateHandlingException;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Validator\ConstraintViolationInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+final class ProductTranslationUpdateListener
```

Comment:
Change this class to a subscriber instead of listener

Reply:
Isn't it a Listener ? `doctrine.event_listener` service

Reply:
Yes, it's a listener. Can you change it to a subscriber? :D